spec/messenger_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'textmessenger'
+
+describe Messenger do
+
+  context "#send_text" do
+
+    it 'sends a payment confirmation text message' do
+      allow(subject).to receive(:send_text)
+      expect(subject).to receive(:send_text).with("Thank you for your order: £27.50")
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)